package.json

```
@@ -80,7 +80,7 @@
   "dependencies": {
     "lodash": "^4.17.15",
     "prop-types": "^15.7.2",
-    "react": "^15.5.0",
-    "react-dom": "^15.3.2"
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
```

I'm now thinking a better and safer approach is to use a wider range here:
```suggestion
    "react": ">=15 <=16",
    "react-dom": ">=15 <=16"
```

This will mean we're fine with any version of react that the user brings in in those ranges.

So if someone uses this lib and is on 15 it will use 15, and otherwise 16. Of course this assumes this lib remains compatible with 15, but maybe we can seperate the update from the deprecation (maybe bump this to 2.0?)

---

maybe better to just go with 16 and if anyone complains (which is unlikely) use this trick...

---

Sounds ok to me, but should there be any usages of React@15?
Bumping the version to 2.0 - will force many users to actively update the version, which we'll probably wouldn't want the overhead of

---

We can add a major version `1.1.0` which would give anyone the option to opt out by `~1.0.0`

---

Yeah, I think let's just go with your original commit and if someone has a problem we can try to solve it :)